cumulusci/tasks/datadictionary.py

```diff
@@ -434,18 +527,24 @@ def _process_field_element(self, sobject, field, version):
                 # Note: polymorphic custom fields are not allowed.
 
             fd = FieldDetail(
-                version,
-                sobject,
-                field_name,
-                field.label.text,
-                f"{field_type}{length}",
-                help_text_elem.text if help_text_elem else "",
-                description_text_elem.text if description_text_elem else "",
-                valid_values,
+                version=version,
+                sobject=sobject,
+                api_name=field_name,
+                label=field.label.text,
+                type=f"{field_type}{length}",
+                help_text=help_text_elem.text if help_text_elem else "",
+                description=description_text_elem.text if description_text_elem else "",
+                valid_values=valid_values,
             )
             fully_qualified_name = f"{sobject}.{fd.api_name}"
             self.fields[fully_qualified_name].append(fd)
 
+    @staticmethod
```

Comment:
What's the reason for this being static? It looks like we already have an instance of the task when these calls are being made.

Reply:
It's not really _needed_ and perhaps is more confusing than it is helpful; it's static simply because it does not access the task's state.